twitter.css

```diff
@@ -0,0 +1,316 @@
+*{
+    box-sizing: border-box;
+}
+body{ margin: 0;
+}
+h2{
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+/* LAY OUT OF PAGE*/
+.main-grid {
+    display: grid;
+    grid-template: ". centered ." auto / 1fr auto 1fr;
```

Comment:
Very good for giving two spaces on the left and right sides `. centered . `
This method works. However, you can also use the 'centre wrapper' to obtain the same result faster and easily compared to the method you used.